### ISSUE TITLE
Mod disappears when loading saved games

### ISSUE THREAD

**Issue by bszonye** [bszonye/civ6-maptacks#29]
A few subscribers have reported problems when continuing a saved game:

> **View The Phenom:** Does anyone else experience an issue where a game does not save the "Map Tacks" mod under additional content? So when continuing a game the mod isn't loaded?

> **BLÀde:** it was working until the sep 3 update now they don't show up at all.

> **kevblackwel:** it doesn't work when you load a game :)

View The Phenom said that they had similar problems with other mods on the [Clean Interface](http://steamcommunity.com/sharedfiles/filedetails/?id=1126873532) list. BLÀde said that the problem went away after unsubscribing and re-subscribing to Map Tacks.

Altogether, this sounds like the 0.9.1 update does not reliably ensure compatibility with conflicting mods like NQ & CQUI. It would help to know what other mods are present in the conflicting games.

**Comment by bszonye** [comment]
If you are seeing this problem in your saved games, please leave a comment and state:

* which mods are in your saved game’s Additional Content list
* which mods are enabled in the Additional Content main menu
* what’s broken or missing – the extra icons, the editor, the scrollbar, etc.

In particular, please note if your game is _unplayable_ – for example, if all map pins disappear from your game entirely, if you cannot see the list of pins, or if you cannot edit pins.

**Comment by bszonye** [comment]
Based on [this CivFanatics thread](https://forums.civfanatics.com/threads/change-to-modding-from-summer-patch.620426/), they may have changed the way `LoadOrder` works, and I might also need to add `priority` attributes to my files.

**Comment by bszonye** [comment]
> **View The Phenom:** For anyone wondering what my issue was, I narrowed it down to JFD's Nubia Civilization pack. If you're using that, you'll need to disable until the conflict is resolved in that mod.

Oddly, JFD’s [Leader Pack: Nubia](https://steamcommunity.com/sharedfiles/filedetails/?id=1099904605) does not even touch the map pin scripts. It does specify a LoadOrder though, which could cause problems post-Summer 2017 update.

**Comment by bszonye** [comment]
See [Mod, Component, & File Load Order as of Winter Patch](https://forums.civfanatics.com/threads/mod-component-file-load-order-as-of-winter-patch.608496/) for more details about `LoadOrder` and `priority` – my current suspicion is that the scripts might be breaking because of the dependencies between the base game scripts and `maptacks.lua`, which they depend on.

**Comment by bszonye** [comment]
Also, it's possible that the `ImportFiles` needs an `InGame` context:

    <ImportFiles id="MAPTACKS_FILES">
      <Properties>
        <LoadOrder>12000</LoadOrder>
        <Context>InGame</Context>
      </Properties>
      <File>maptacks.lua</File>
      ...
    </ImportFiles>

**Comment by bszonye** [comment]
BLÀde provided a mod list to check for conflicts:

* better trade screen
* furthur trade routes
* gdg no penalty different governments
* gold resource
* improved fog of war
* less gossip 1.0
* more lenses
* plots roads and mnovement v 1.02
* production queue
* removable districts
* resourceful
* show diplomatic deals v1.05
* starting scout
* stop steam rolling citty states
* your improved mod screen
* toms history relgions
* unit report screen
* further zoom

Some of these may only be available at CivFanatics.

**Comment by bszonye** [comment]
On CivFanatics, [LeeS discovered](https://forums.civfanatics.com/threads/problems-resolving-mod-conlicts.622065/#post-14859190) that [Leader Pack: Nubia](https://steamcommunity.com/sharedfiles/filedetails/?id=1099904605) by JFD conflicts with *all* mods with AffectsSavedGames=0.

**Comment by bszonye** [comment]
I tracked down the Leader Pack: Nubia problem to one of its dependencies:

    <Dependencies>
      <Mod id="643EA320-8E1A-4CF1-A01C-00D88DDD131A" title="Nubia_Amanitore" />
      <Mod id="7619F77B-BAA7-4912-AA6E-45C124E6ECFD" title="Nubia Scenario" />
    </Dependencies>

When I remove the second dependency (Nubia Scenario), then all the other mods load normally.

It looks like I cannot directly fix the bug, but I notified JFD and recommended a fix. If they do not fix the mod before the next Map Tacks release, I can set a block rule to make the mods mutually exclusive.
